src/template_generation_tools.py

```diff
@@ -58,8 +60,8 @@ def generate_class_graph_template(ccf_tools_df :pd.DataFrame):
  
   # Add declarations and labels for entity
   for i, r in ccf_tools_df.iterrows():
-    records.append({'ID': r['s'], 'User_label': r['user_slabel']})
-    records.append({'ID': r['o'], 'User_label': r['user_olabel']})
+    records.append({'ID': r['s'], 'User_label': r['user_slabel'], 'present_in_taxon': 'NCBITaxon:9606', 'in_subset': 'HubMAP_ASCT'})
```

Comment:
`present_in_taxon` and `in_subset` not generating correcly in OWL. Any ideia why?